sdk/core/System.Net.ClientModel/src/ModelReaderWriter/IModel.cs

```diff
@@ -32,6 +32,6 @@ public interface IModel<out T>
         /// <param name="options">The <see cref="ModelReaderWriterOptions"/> to use.</param>
         /// </summary>
         /// <returns>The <see cref="ModelReaderWriterFormat"/> that the model uses for its wire format.</returns>
-        ModelReaderWriterFormat GetWireFormat(ModelReaderWriterOptions options);
+        Type GetInterfaceType(ModelReaderWriterOptions options);
```

Comment:
Curious what this will do - (I'll dig around in the samples to see if I can figure it out) ... does it return IModel if wire format is not JSON and IJsonModel if it is JSON?